city_scrapers/spiders/stl_forest_park_advisory.py

```diff
@@ -0,0 +1,185 @@
+            url=agenda_url, method="GET", callback=self._get_agenda_urls, priority=100
+        )
+        yield scrapy.Request(
+            url=agenda_url, method="GET", callback=self._get_minute_urls, priority=100
+        )
+
+        meeting_urls = [
+            (
+                "https://www.stlouis-mo.gov/government/departments"
+                "/parks/parks/forest-park-advisory-board.cfm"
+            ),
+            ("https://www.stlouis-mo.gov/events/past-meetings.cfm?department=117"),
+        ]
+        for url in meeting_urls:
+            yield scrapy.Request(url=url, method="GET", callback=self.parse)
+
+    def parse(self, response):
+        for url in self._get_event_urls(response):
+            yield scrapy.Request(url=url, callback=self._parse_event)
+
+    def _get_event_urls(self, response):
+        event_titles = response.xpath(
+            '//*/a[contains(@href, "Event_ID")]/strong/text()'
+        ).getall()
+        if len(event_titles) == 0:
+            event_titles = response.xpath(
+                '//*/a[contains(@href, "Event_ID")]/text()'
+            ).getall()
+        event_urls = response.xpath('//*/a[contains(@href, "Event_ID")]/@href').getall()
+        urls = []
+        for url, title in zip(event_urls, event_titles):
+            if "Forest Park Advisory" in title:
+                urls.append(response.urljoin(url))
+        return urls
+
+    def _parse_event(self, response):
+
+        start = self._parse_start(response)
+        links_key = datetime.strftime(start, "%Y-%m-%d")
+
+        meeting = Meeting(
+            title=self._parse_title(response),
+            description=self._parse_desc(response),
+            classification=BOARD,
+            start=start,
+            end=self._parse_end(response),
+            all_day=False,
+            location=self._parse_location(response),
+            source=response.url,
+        )
+
+        meeting["links"] = []
+
+        if links_key in self.agenda_map.keys():
+            meeting["links"].append(self.agenda_map[links_key])
+
+        if links_key in self.minute_map.keys():
+            meeting["links"].append(self.minute_map[links_key])
+
+        return meeting
+
+    def _parse_start(self, response):
+        date = response.xpath('//*/p[@class="page-summary"]/text()').get()
+        date = date.replace("\n", "")
+        pattern = r"(?P<day>\d{2}/\d{2}/\d{2}), (?P<time>(\d{1,2}:\d{2}) (PM|AM))"
+        pattern += r" - (\d{1,2}:\d{2}) (PM|AM)"
+        rm = re.search(pattern, date)
+
+        if rm is not None:
+            day = rm.group("day")
+            time = rm.group("time")
+            dt = day + " " + time
+            start = datetime.strptime(dt.strip(), "%m/%d/%y %H:%M %p")
+            return start
+        else:
+            return None
+
+    def _parse_title(self, response):
+        title = response.xpath('//*/div[@class="EventTitle"]/text()').get()
+        title = title.replace("\n", "")
+
+        return title
```

Comment:
This may not be necessary, since all of the titles are "Forest Park Advisory Board".